comp110_architecture.tex

```diff
@@ -0,0 +1,80 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{An Analysis of the Variability, Reliability, and Control Provided by Level Generation Techniques for 2D Platform Games}
+
+% Please do not change the subtitle
+\subtitle{COMP110 - Computer Architecture Essay}
+
+% Please put your student number in the author field
+\author{1507290}
+
+\begin{document}
+
+\maketitle
+
+\abstract{There are a number of diverse approaches to level generation for 2D platform games, each placing emphasis on a different aspect of level generation. Chunk-based approaches such as Occupancy Regulated Extension tend to favour control and variation, whilst rhythm-based approaches such as that demonstrated by Launchpad favour playability and enjoyability. There are also evolutionary approaches that use genetic algorithms, which when combined with constraint satisfaction, as demonstrated by Sorenson et al., provide a high level of control and reliability.}
```

Comment:
I would not mention names of other authors (or cite work) in abstracts. I would remove the "as demonstrated by" clause.